### PR TITLE
Adjust LPA cost from 110 GBP to 82 GBP

### PR DESCRIPTION
**Pull request by taylorsj1980**

### CODE REVIEW

File: src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php [hunk 1]
```diff
@@ -5,61 +5,53 @@
...
-        if(($lpa->payment->reducedFeeReceivesBenefits) && ($lpa->payment->reducedFeeAwardedDamages)) {
+        if (!$lpa->payment instanceof Payment) {
+            return null;
+        }
 
+        if ($lpa->payment->reducedFeeReceivesBenefits && $lpa->payment->reducedFeeAwardedDamages) {
             $amount = self::getBenefitsFee();
-
         } else {
+            $isRepeatApplication = ($lpa->repeatCaseNumber != null);
```
Comment (elvisciotti) [inline src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php]: != would give the unexpected results if the var is zero or false or empty string


$isRepeatApplication will

File: src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php [hunk 2]
```diff
@@ -5,61 +5,53 @@
...
+        return (float) self::getFullFee($isRepeatApplication) / 2;
     }
 
-    public static function getBenefitsFee(){
-
-        return (float)0;
-
+    public static function getBenefitsFee()
+    {
+        return (float) 0;
```
Comment (elvisciotti) [inline src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php]: or 
return 0.0

Comment (taylorsj1980) [inline src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php]: Changed as per comments

File: src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php (hunk 1 above)
Comment (taylorsj1980) [inline src/Opg/Lpa/DataModel/Lpa/Payment/Calculator.php]: As discussed, I will leave this as it is for now. It has been working and changing to a stricter check might cause issues elsewhere.

**Pull request merged**